imapautofiler/actions.py

```diff
@@ -254,6 +254,44 @@ def invoke(self, conn, mailbox_name, message_id, message):
         )
 
 
+class Mark_as(Action):
```

Comment:
Let's stick to names without underscores for classes. If this is flagging, we could call it Flag. If it's more general we could just call it Mark. Then the other class could be Unflag or Unmark.